src/CheckoutSdk/ICheckoutApiClient.cs

```diff
@@ -0,0 +1,9 @@
+namespace Checkout
+{
+    /// <summary>
+    /// Marker interface
+    /// </summary>
+    public interface ICheckoutApiClient
+    {
+    }
+}
```

Comment:
This Interface is empty, I don't understand why?

Reply:
I left a comment on the top. This is a marker interface I forcibly created to make use of generics at `CheckoutServiceCollection`. Both `ICheckoutApi` are inheriting this interface, so we do something like: `where TC : ICheckoutApiClient`. This is the usage you are able to find in this PR:

```
private static void SetCommonAttributes<TB, TC>(TB builder, CheckoutOptions options,
    ILoggerFactory loggerFactory,
    IHttpClientFactory httpClientFactory) where TB : AbstractCheckoutSdkBuilder<TC>
    where TC : ICheckoutApiClient
{
```
Since we are using generics at `AbstractCheckoutSdkBuilder` for `ICheckoutClient` and `Four.ICheckoutClient`, this way I can say that that method can accept any `AbstractCheckoutSdkBuilder` that works on a type that extends `ICheckoutApiClient`.